### ISSUE TITLE
Is there an edit-page-by-form plugin

### ISSUE THREAD

**Issue by jacmgr** [bleutzinn/grav-plugin-add-page-by-form#18]
This not issue, but humble request.  Did not know how to mark this as request.  I know there is the "grav-plugin-editable" which does what it does wonderfully.  I am trying to learn to write my own plugins, but I am not very smart enough yet in php or grav to follow the "editable" plugin's code though!!  However, this "edit-by-form" plugin seemed to me to be so much more straightforward and concise, and I could grasp all the code and what it accomplishes; and it also does exactly what it says it does wonderfully.   If there was such a simple way to do the same thing for "editing an existing page by form plugin", I think I could learn a lot about grav and php.  Would you have some suggestions on how to do that? I'd like to try.  (Sorry if this not the correct forum to discuss this.)

**Comment by bleutzinn** [comment]
Issues can be anything, a problem or bug report, a feature request or a question. No problem.

The Editable plugin is relatively complex and it took me quite some time to understand Grav and existing plugins to create it. Editable uses parts of other plugins and is dependent upon the Shortcode Core plugin. Just like the Add Page plugin depends on the Form plugin. The best way to learn is to try.

One idea you might try is to pre-fill the content of the SimpleMDE editor with the Grav page content you want to edit. See [Get/set the content](https://github.com/NextStepWebs/simplemde-markdown-editor#getset-the-content) in the SimpleMDE documentation. This involves more Javascript than PHP but I think it is a relatively easy and effective way to do what you want.

**Comment by bleutzinn** [comment]
If I can help, let me know. I'm closing the issue for now.

**Comment by jacmgr** [comment]
Closed is fine! Thankyou.  Still working it. Turns out Pre-Filling a standard Grav Form field's default value from a plugin is not supported ?apparently?.   I have a question posted at the form plugin repo.  In the meantime, I have progressed on a different track of basically creating plugin with its own form templates (like login plugin); in plugin twigvars events set a twig variable for pagecontent to be default in the text area; and then in the plugin's form templates use that pagecontent twig var in the hardcoded html for the text area.    Draft proof of concept is working.  I'll let you know how it all turns out.
